.github/actions/setup-linux/action.yml

```diff
@@ -29,10 +29,12 @@ runs:
     - name: Start docker if docker deamon is not running
       shell: bash
```

Comment:
Perhaps we can add : ``! -e /.inarc`` as ENV variable and then this step could be skipped:
``
if: ${{ ! IN_ARC }}
``

Reply:
good point, using `if: ${{ hashFiles('/.inarc') == '' }}` seems even better, as syntax beautifulness is concerned